var/spack/repos/builtin/packages/hybrid-lambda/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HybridLambda(AutotoolsPackage):
+    """Hybrid-Lambda is a software package that can simulate gene trees
+       within a rooted species network or a rooted species tree under the
+       coalescent process. The main feature of this program is that users can
+       choose to use the standard Kingman coalescent process, which produces
```

Comment:
If you leave an empty line and start with a short paragraph you'll get both a short and a long description of the software in different contexts:
```suggestion
    """Hybrid-Lambda is a software package that can simulate gene trees
       within a rooted species network or a rooted species tree under the
       coalescent process. 
       
       The main feature of this program is that users can
       choose to use the standard Kingman coalescent process, which produces
```

Reply:
Thanks for the suggestion!